[2025-03-26T00:52:39.353+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: titanic_survival_classifier.model_validation_task manual__2025-03-26T00:17:50.145574+00:00 [queued]>
[2025-03-26T00:52:39.370+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: titanic_survival_classifier.model_validation_task manual__2025-03-26T00:17:50.145574+00:00 [queued]>
[2025-03-26T00:52:39.371+0000] {taskinstance.py:2170} INFO - Starting attempt 4 of 4
[2025-03-26T00:52:39.399+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): model_validation_task> on 2025-03-26 00:17:50.145574+00:00
[2025-03-26T00:52:39.410+0000] {standard_task_runner.py:60} INFO - Started process 1866 to run task
[2025-03-26T00:52:39.415+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'titanic_survival_classifier', 'model_validation_task', 'manual__2025-03-26T00:17:50.145574+00:00', '--job-id', '22', '--raw', '--subdir', 'DAGS_FOLDER/pipeline.py', '--cfg-path', '/tmp/tmpv9elejf1']
[2025-03-26T00:52:39.417+0000] {standard_task_runner.py:88} INFO - Job 22: Subtask model_validation_task
[2025-03-26T00:52:39.441+0000] {warnings.py:109} WARNING - /home/***/.local/lib/python3.8/site-packages/***/settings.py:194: DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
  SQL_ALCHEMY_CONN = conf.get("database", "SQL_ALCHEMY_CONN")

[2025-03-26T00:52:39.510+0000] {task_command.py:423} INFO - Running <TaskInstance: titanic_survival_classifier.model_validation_task manual__2025-03-26T00:17:50.145574+00:00 [running]> on host 0f3b2e2ddf0a
[2025-03-26T00:52:39.682+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='titanic_survival_classifier' AIRFLOW_CTX_TASK_ID='model_validation_task' AIRFLOW_CTX_EXECUTION_DATE='2025-03-26T00:17:50.145574+00:00' AIRFLOW_CTX_TRY_NUMBER='4' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-03-26T00:17:50.145574+00:00'
[2025-03-26T00:52:39.738+0000] {logging_mixin.py:188} INFO - ---Testing Model---
[2025-03-26T00:52:39.758+0000] {logging_mixin.py:188} INFO - Test predictions complete
[2025-03-26T00:52:39.777+0000] {logging_mixin.py:188} INFO - Submission saved at /opt/***/data/submission/submission.csv
[2025-03-26T00:52:39.778+0000] {python.py:201} INFO - Done. Returned value was: {'submission_file': '/opt/***/data/submission/submission.csv', 'test_predictions': [0, 0, 0, 0, 1, 0, 1, 0, 1, 0, 0, 0, 1, 0, 1, 1, 0, 0, 1, 0, 0, 0, 1, 1, 1, 0, 1, 0, 0, 0, 0, 0, 0, 0, 1, 0, 1, 1, 0, 0, 0, 1, 0, 1, 1, 0, 0, 0, 1, 0, 1, 0, 1, 1, 0, 0, 0, 0, 0, 1, 0, 0, 0, 1, 1, 1, 1, 0, 0, 1, 1, 0, 1, 1, 1, 1, 0, 1, 0, 1, 0, 0, 0, 0, 0, 0, 1, 1, 1, 0, 1, 0, 1, 0, 0, 0, 1, 0, 1, 0, 1, 0, 0, 0, 1, 0, 0, 0, 0, 1, 0, 1, 1, 1, 1, 0, 0, 1, 0, 1, 1, 0, 1, 0, 0, 1, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 1, 0, 0, 1, 0, 0, 0, 0, 1, 0, 0, 0, 1, 0, 0, 0, 0, 0, 1, 1, 0, 1, 1, 0, 1, 0, 0, 1, 0, 0, 1, 1, 0, 0, 0, 0, 0, 1, 1, 0, 1, 1, 0, 1, 1, 0, 1, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 0, 1, 1, 0, 1, 1, 0, 1, 1, 0, 0, 1, 0, 1, 0, 0, 0, 0, 1, 0, 0, 1, 0, 1, 0, 1, 0, 1, 0, 1, 1, 0, 1, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 1, 1, 1, 1, 1, 0, 0, 1, 1, 0, 1, 1, 1, 0, 1, 0, 0, 0, 0, 0, 1, 0, 0, 0, 1, 1, 0, 0, 0, 0, 1, 0, 0, 0, 1, 1, 0, 1, 0, 0, 0, 0, 1, 0, 1, 1, 1, 0, 0, 1, 0, 0, 0, 1, 0, 0, 0, 0, 1, 0, 0, 0, 0, 1, 0, 0, 1, 1, 0, 0, 0, 0, 0, 0, 0, 1, 1, 1, 0, 0, 0, 0, 0, 0, 0, 0, 1, 0, 1, 0, 0, 0, 1, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 0, 1, 0, 1, 0, 1, 0, 1, 1, 0, 0, 0, 1, 0, 1, 0, 0, 1, 0, 1, 1, 0, 1, 0, 0, 1, 1, 0, 0, 1, 0, 0, 1, 1, 1, 0, 0, 0, 0, 0, 1, 1, 0, 1, 0, 0, 0, 0, 1, 1, 0, 0, 0, 1, 0, 1, 0, 0, 1, 0, 1, 0, 1, 1, 0, 0, 1, 1, 1, 1, 1, 0, 1, 0, 0, 0]}
[2025-03-26T00:52:39.820+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=titanic_survival_classifier, task_id=model_validation_task, execution_date=20250326T001750, start_date=20250326T005239, end_date=20250326T005239
[2025-03-26T00:52:39.868+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-03-26T00:52:39.903+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
